src/docs.js

```diff
@@ -0,0 +1,174 @@
+import sections from './sections.js'
+
+export default [
+  {
+    name: "Ape Framework",
+    link: "https://docs.apeworx.io/ape/stable",
+    section: sections.CORE
+  },
+  {
+    name: "Silverback",
+    link: "https://docs.apeworx.io/silverback/stable",
+    section: sections.CORE
+  },
+  {
+    name: "Etherscan",
+    link: "https://github.com/ApeWorX/ape-etherscan",
+    section: sections.PLUGIN
```

Comment:
Can a plugin sub classification be added?
- accounts
- compilers
- ecosystems
- providers 
- converters (or other)

Reply:
will make another issue for this